### PR TITLE
Fix typos in ``02_Filterbank.ipynb``

### PR DESCRIPTION
**Pull request by hihunjin**
``asteroid.filterbanks`` to ``asteroid_filterbanks`` on the colab notebook.

Additionally, I changed the link of [asteroid_filterbanks](https://github.com/asteroid-team/asteroid-filterbanks) from [asteroid.filterbanks](https://github.com/asteroid-team/asteroid/tree/master/asteroid/filterbanks).

This PR is with the issue "No module named 'asteroid.filterbanks' #525"

Thanks in advance.

### CODE REVIEW

File: notebooks/02_Filterbank.ipynb [hunk 1]
```diff
@@ -126,8 +126,7 @@
     }
    ],
    "source": [
-    "from asteroid.filterbanks import STFTFB\n",
-    "from asteroid.filterbanks.transforms import take_mag\n",
```
Comment (hihunjin) [inline notebooks/02_Filterbank.ipynb:130]: This line is redundant.

File: notebooks/02_Filterbank.ipynb [hunk 2]
```diff
@@ -193,7 +192,7 @@
    "metadata": {},
    "source": [
     "## Making an encoder-decoder pair in one line \n",
-    "[`make_enc_dec`](https://github.com/asteroid-team/asteroid/blob/master/asteroid/filterbanks/__init__.py#L12) is a small function that helps making encoder-decoder pairs in an efficient way. Let's see some examples :\n"
+    "[`make_enc_dec`](https://github.com/asteroid-team/asteroid-filterbanks/blob/master/asteroid_filterbanks/__init__.py#L14-L70) is a small function that helps making encoder-decoder pairs in an efficient way. Let's see some examples :\n"
```
Comment (hihunjin) [inline notebooks/02_Filterbank.ipynb:195]: to the ``asteroid/asteroid-filterbanks`` repository

**Comment by JorisCos** [comment]
Thanks for the fix, it's quite useful. Can you had the notebook `02_Filterbank` in the PR's title just for the sake of github PR history

**Comment by hihunjin** [comment]
@JorisCos does this look good? Thanks 😊

**Comment by JorisCos** [comment]
Looks great !

**Review by mpariente — APPROVED** [review]
Thanks again @hihunjin

**Pull request merged**